include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -1966,59 +1962,58 @@ class charge_distribution_surface<Lyt, false> : public Lyt
     }
     /**
      *  The stored unique index is converted to a charge distribution.
+     *
+     *  @param incremented Flag that can be set to true when the charge index was incremented before calling this
+     * function to enable an optimization. When set to true, the leading zeroes of the charge index are not handled.
```

Comment:
```suggestion
     *  @param incremented Flag that can be set to `true` if the charge index was incremented right before calling this
     * function to enable an optimization. When set to `true`, the leading zeroes of the charge index are ignored.
```

Reply:
Please adjust accordingly if you agree to change the argument type of this function.

Reply:
#347 implements this change